### DISCUSSION TITLE
Help installing on rasbenOS

### DISCUSSION THREAD

**Discussion by Josh65-2201** [wwmm/easyeffects#2623] [Q&A]
There isn't a easyeffects package on the native package repository program and I can't find any working Instructions on the github or google.

**Comment by vchernin** [comment]
I'm not sure what rasbenOS is, but you may be able to use the flatpak package.

**Reply by Josh65-2201** [reply]
It's this https://www.raspberrypi.com/software/. Seem they have changed the name.

**Reply by vchernin** [reply]
Then it should be possible to use flatpak. Make sure you are using 64-bit raspberry pi os https://flathub.org/setup/Raspberry%20Pi%20OS

**Reply by Josh65-2201** [reply]
So it installed but won't open. Running it from the program's panel it's showing the loading cursor for a bit then goes back to the normal one with no window opening. From the terminal it shows
'Pipe_manager.cpp: 1219 an error occurred while retrieving the object serial. This port cannot be handled by easy effects.' Also with no window open.

**Comment by wwmm** [comment]
>From the terminal it shows
'Pipe_manager.cpp: 1219 an error occurred while retrieving the object serial. This port cannot be handled by easy effects.' Also with no window open.

This error alone is not enough to avoid the window opening. Unless your PipeWire installation is so broken EasyEffects gets stuck at startup. Check if you have all PipeWire packages installed and that Pulseaudio is not running.

It will probably help to follow this page's [step number 3](https://github.com/wwmm/easyeffects/wiki/Reporting-Bugs#basic-logs) to run in debug mode.

**Reply by Josh65-2201** [reply]
Hello, running debug mode shows the below with the warning repeating for some time
![image](https://github.com/wwmm/easyeffects/assets/26977075/4f1d869e-df75-4df9-aaed-6d8b9bb320a3)

**Reply by wwmm** [reply]
It is definitely not normal to have so many warnings of this kind. I wonder if it is related to the fact your system seems to have PipeWire `0.3.19`. Under the hoods PipeWire changed a lot between `0.3.19` and `0.3.69`. I am afraid EasyEffects may not be compatible with 0.3.19 anymore.

**Reply by wwmm** [reply]
At this moment Arch Linux has PipeWire `0.3.80`. I think rasbenOS folks need to update PipeWire more frequently.
